apps/web/src/pages/SearchRequests/SearchPage/hooks.ts

```diff
@@ -74,22 +69,14 @@ export const useCandidateCount = (
     [filters],
   );
 
-  // The countApplicants query ignores the pool filter if it is an empty array, just like if it were undefined.
-  // However, we want to treat an empty pool filter as resulting in zero candidates.
-  // Therefore, we can skip the query and override the count results ourselves.
-  const hasPools =
```

Comment:
How does removing this change the results?

Reply:
This was preventing the initial query to get the count from firiring until we had all the published pools since otherwise it would retunr unexpected results.

Since we no longer use pools, it is no longer needed and we don't need to pause the query and just let it run on load now :+1: